scipy/spatial/tests/test_spherical_voronoi.py

```diff
@@ -166,7 +166,7 @@ def test_sort_vertices_of_regions_flattened(self):
         expected = sorted([[0, 6, 5, 2, 3], [2, 3, 10, 11, 8, 7], [0, 6, 4, 1],
                            [4, 8, 7, 5, 6], [9, 11, 10], [2, 7, 5],
                            [1, 4, 8, 11, 9], [0, 3, 10, 9, 1]])
-        expected = list(itertools.chain(*sorted(expected)))
+        expected = list(itertools.chain(list(*sorted(expected))))
```

Comment:
needed because: `scipy/spatial/tests/test_spherical_voronoi.py:169: error: Argument 1 to "chain" has incompatible type "*List[List[int]]"; expected "Iterable[List[int]]"  [arg-type]`

Reply:
Revised to ignore the typing on this statement for now since the change wasn't functionally equivalent; `mypy` bug maybe?